lib/index.js

```diff
@@ -90,8 +90,43 @@ MapboxGeocoder.prototype = {
     }
   },
 
+  /**
+   * Add the geocoder to a container. The container can be either a mapboxgl.Map or a reference to an HTML class or ID. 
+   * 
+   * If the container is a mapboxgl.Map, this function will behave identically to Map.addControl(geocoder)
+   * If the container is an html id or class, the geocoder will be appended to that element
+   * 
+   * This function will throw an error if the container is not one a map or a class/id reference
+   * It will also throw an error if the referenced html element cannot be found in the document.body
+   * @param {String|mapboxgl.Map} container A reference to the container to which to add the geocoder 
+   */
+  addTo: function(container){
+    // if the container is a map, add the control like normal
+    if (container._controlContainer){
+      //  it's a mapbox-gl map, add like normal 
+      container.addControl(this)
+    }
+    // if the container is not a map, but an html element, then add the control to that element
+    else if (typeof container == 'string' && (container.startsWith('#') || container.startsWith('.'))){
+      const parent = document.querySelectorAll(container);
+      if (parent.length == 0){
+        throw new Error("Element ", container, "not found.")
+      }
+
+      parent.forEach(function(parentEl){
```

Comment:
I was able to create two geocodes with the same class applied to two different elements, but they behave unpredictably once created 
![gif](https://cl.ly/29c53110b723/Screen%252520Recording%2525202019-09-27%252520at%25252004.12%252520PM.gif)